### ISSUE TITLE
CSS text content is not output correctly (Chrome, IE 11)

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#297]
### Summary

CSS text content is not output correctly (Chrome, IE 11)

1. save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>CSS content</title>
		<style>
			.text::before {content:"before"}
			.text::after {content:"after"}
		</style>
	</head>
	<body>
		<p class=text>1</p>
		<a href=# class=text>2</a>
		<button class=text>3</button>
		<fieldset>
			<legend class=text>4</legend>
			<label class=text>5 <input></label>
			<label class=text for=6>6</label> <input id=6>
		</fieldset>
	</body>
</html>
```
2. read with arrow keys
3. navigate with Tab key

### Expected result
* Consistent output between browsers
* CSS content is output

### Actual result 
* IE 11: no output
* Firefox: is output correctly
* Chrome: only output sometimes:
	* button, legend: no output
	* label: Output only when reading with the arrow keys
	* text, link: correct output

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431

**Comment by stevefaulkner** [comment]
> IE 11: no output
not expected as IE11 does not expose content in the acc tree.

also
[test case](https://codepen.io/stevef/pen/MWwwJWb)

**Comment by carmacleod** [comment]
This is most likely due to [Chrome bug 1130491](https://bugs.chromium.org/p/chromium/issues/detail?id=1130491).

**Comment by JAWS-test** [comment]
@carmacleod Yes, that's my guess as well. What is strange, though, is that even with the link and button, only the text content and not the CSS pseudo-content is transmitted by Chrome. Nevertheless, JAWS outputs the link correctly, i.e. completely, and NVDA outputs the link and the button correctly. I.e. there are elements where the Chrome error has no effect on the screen reader output

**Comment by carmacleod** [comment]
Hmmm, yep, that's strange. :)
Possibly both JAWS and NVDA are trying to work around the Chrome issue, but JAWS missed the button use case.
